src/en/space-station-14/departments/service/proposals/gnome-ghost-role.md

```diff
@@ -0,0 +1,85 @@
+ - very low health, if caught annoying crew or stealing or other activities they can be easily killed by whatever is on hand 
+ - no inventory, everything must be held and containers cannot be dragged under doors meaning major theft has to be planned in advance 
+ - cannot use guns 
+
+Gnomes also may get annoying and demand server resourses if there are enough of them. currently 6 gnomes is the ideal normal max, although if botany tries very very 
+hard they may make more than six. the idea is one Gnome per hat. being this limited however it would be simple for a greytide to round remove all of the gnomes. 
+gnomes spawning a plantable gnome shard on death helps prevent gnomes from being removed from the round entirely.
+The gnome hat will spawn as maints loot. 
+Gnomes also may give a larger insentive to lock lockers so gnomes do not pilfer items (more important with syndicate gnomes as gnomes should NOT have an antag pass, 
+petty crime is fine but sneaking into atmos and plasma flooding is NOT)
+Gnomes are also grouned in humor, the limitations on them (low health and one hand with no inventory) would inspire ways to work around things. 
+They have oportunities for strange gimmics but not much for anything actively harmfull. some examples of this Gremlin embracing idea are: 
+hoarding, trading, stealing, fort construction, gnome maints bar, and so on. 
+
+Idealy gnomes are useful helpers at best and crafty pests at worst
+
+## traitor gnomes and non standard gnomes
+
+After the helper gnome is implimented I will begin working on some of the following features to be added seprately.
+
+Gnome types after The basic gnome are not very well thought out and will need a good amount of thought put into them. 
+As it is thought of now a Trator will be able to call in a Gnome renforcement that a player can inhabit. they will have a specific red sprite to diferentiate gnome types.
+the primary role of this gnome would be to do things stealthily for the agent. Gnomes and Syndicate gnomes would be given headset slots if this was implimented. they synicate gnome would keep the same 10 health and 
+the headset and the gnome can be captured (gnomes do not require air thus they can be locked in a evidence locker or simply killed) 
+Idealy this would add another layer onto stealth options for any agent, from gnome guided bombs (the gnome may refuse to be a bomb, that is up to the gnome) 
+to a part of a complex planned heist. 
+
+other types of gnomes would be mutated gnomes, using mutagen on a gnome plant would have a chance to mutate the plant into differnt types of gnome, 
+be that a gnome that spills space lube when it dies, larger stronger gnomes, or gnomes that glow in the dark. 
+
+additionaly gnomes have a potential for interesting mid round events. one example is gnomes that spawn out of vents or a gnome king that wishes to convert botany into a 
+gnome kingdom to grow more gnomes (these would act like rat servants) 
```

Comment:
this is a little bloaty since i dont think traitors will buy this as they can achieve the same with other tools, gnomes have some potential for expansion in the future but that is something that should be added to the doc later. So prob best to delete this section